client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -721,6 +721,9 @@
                     $scope.itemActions = null;
                     if (new_value) {
                         $scope.itemActions = authoring.itemActions(new_value);
+                        desks.fetchDeskById(new_value.task.desk).then(function (desk) {
```

Comment:
imo it would be better if it's somewhere nearby stage handling. and probably both could be in a directive